package.json

```diff
@@ -197,6 +197,7 @@
     "emoji-regex": "10.4.0",
     "emojibase": "16.0.0",
     "emojibase-regex": "16.0.0",
+    "escape-string-regexp": "4.0.0",
```

Comment:
like other nodejs package of sindresorhu, escape-string-regexp@4 is cjs package and escape-string-regexp@5 is esm package

Reply:
drop it, see other comment. 😉